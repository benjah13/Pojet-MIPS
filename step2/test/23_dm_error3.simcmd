dm 0xza # 'za' n'est pas en base 16
