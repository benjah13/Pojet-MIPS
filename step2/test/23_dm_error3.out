Invalid param : hexadecimal number awaited in 'za'
